### PR TITLE
Remove various mentions of Windows for server usage

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/SsE1sGqR/33-in-place-import-doc-is-unclear

Not sure how we want to handle dropbox, shall I remove all mention of Windows support there too @joshmoore ?

### CODE REVIEW

**Comment by joshmoore** [comment]
I'd assume dropbox as well.

File: omero/sysadmins/server-advanced-configuration.rst [hunk 1]
```diff
@@ -7,9 +7,7 @@ In some cases, the configuration properties will not suffice to fully
 configure your server. In that case, it may be necessary to make use of
 IceGrid's XML configuration files. Like the :file:`config.xml` file mentioned
 above, these are stored under :file:`etc/grid`:
-:file:`etc/grid/default.xml` is used on Unix systems,
-:file:`etc/grid/windefault.xml` is used on
```
Comment (mtbc) [inline omero/sysadmins/server-advanced-configuration.rst:11]: @joshmoore: Should we get rid of this file?

**Review by mtbc — APPROVED** [review]

File: omero/sysadmins/server-advanced-configuration.rst (hunk 1 above)
Comment (joshmoore) [reply omero/sysadmins/server-advanced-configuration.rst:11]: I'd think so.

Comment (mtbc) [reply omero/sysadmins/server-advanced-configuration.rst:11]: Created https://trello.com/c/Zj6iJqHc/780-remove-traces-of-windows-server - I might have missed something related.

**Pull request merged**